.circleci/config.yml

```diff
@@ -1,827 +1,51 @@
 version: 2.1
 
 references:
-  - &OTP24 cimg/elixir:1.12  # Contains Erlang 24.0
+  - &CERT_KEY certs-cache-{{ checksum "tools/ssl/Makefile" }}-v2
```

Comment:
This would then pretty much generate the same key forever, and we'll fall into the previous trap I felt for in master.

Reply:
yea, would get expired in 2 weeks (so resolved though slack conversation)